.github/workflows/testsuite.yml

```diff
@@ -74,23 +74,14 @@ jobs:
           # python -c "import urllib.request as r; exec(r.urlopen('https://bit.ly/pydoctor').read())"
 
       - name: "Run tox for ${{ matrix.python-version }}"
-        continue-on-error: true
-        id: tox1
         run: |
           python -m tox -- -rfsEX
 
       - name: "Retry tox for ${{ matrix.python-version }}"
-        id: tox2
-        if: steps.tox1.outcome == 'failure'
-        run: |
-          python -m tox -- -rfsEX
-
-      - name: "Set status"
-        if: always()
-        run: |
-          if ${{ steps.tox1.outcome != 'success' && steps.tox2.outcome != 'success' }}; then
-             exit 1
-          fi
+        if: failure()
+        run: >-  # `exit 1` makes sure that the job remains red with flaky runs
+          python -m tox -- -rfsEX --lf -vvvvv
```

Comment:
@nedbat did I get it right that the posargs are being forwarded to pytest?

Reply:
yes

Reply:
Then, this should work as intended. I've started practicing this approach relatively recently and it works well.